internal/k8shandler/nodetypefactory.go

```diff
@@ -19,6 +19,7 @@ type NodeTypeInterface interface {
 	isMissing() bool
 	name() string
 	delete() error
+	GetSecretHash() string
```

Comment:
can we change the scope of this function? `getSecretHash() string` or does that cause this change to no longer work?